servers/audio/effects/audio_effect_pitch_shift.h

```diff
@@ -77,7 +77,7 @@ class AudioEffectPitchShift : public AudioEffect {
 public:
 	friend class AudioEffectPitchShiftInstance;
 
-	enum FFTSize {
+	enum FFTSize : unsigned int {
```

Comment:
Intentionally using `unsigned int` here over `uint32_t` or `uint64_t` because this is old code that still use such "old" types like `long`. Refactoring it is out of scope for this quick fix.